api-samples/override/override_igoogle/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Thoughts on deleting this sample? It basically shows the same thing as the other one, and links to iGoogle which doesn't exist anymore, so I don't think it adds much.

Reply:
+1